main.go

```diff
@@ -28,12 +28,21 @@ func (s Service) AddressAndPort() string {
 
 var timeout = flag.Int64("timeout", 60, "time to wait for all services to be up (seconds)")
 var httpPort = flag.Int("httpport", 0, "wait for an http request if target port is given port")
-var ignorePort = flag.Int("ignoreport", 0, "don't wait for services on this port to be up")
+var _ignorePort = flag.String("ignoreport", "", "don't wait for services on these ports to be up (separated by comma)")
 
 func main() {
 	setupUsage()
 	flag.Parse()
 
+	var ignorePortStr = strings.Split(*_ignorePort, ",")
+	var ignorePorts []int
+
+	for _,portStr := range ignorePortStr {
+	  	if s, err := strconv.ParseInt(portStr, 10, 64); err == nil {
```

Comment:
Also, please handle possible errors.

Thinking about error handling, it might be nicer to use a custom type with [`flag.var`](https://golang.org/pkg/flag/#Var). This would integrate error handling with `flag.Parse()` and would move the parsing out of the main function, which is large enough anyway.